boards/kyria/kb_v2.py

```diff
@@ -0,0 +1,67 @@
+from kmk.kmk_keyboard import KMKKeyboard as _KMKKeyboard
+from kmk.quickpin.pro_micro.sparkfun_promicro_rp2040 import pinout as pins
+from kmk.scanners import DiodeOrientation
+from kmk.scanners import intify_coordinate as ic
+from storage import getmount
+
+isLeftSide = False if str(getmount('/').label)[-1] == 'R' else True
+leftColPins = (
+        pins[6],
+        pins[7],
+        pins[8],
+        pins[9],
+        pins[10],
+        pins[11],
+        pins[12],
+        pins[13],
+    )
+leftRowPins = (pins[17], pins[16], pins[15], pins[14])
+
+rightColPins = (
+        pins[10],
+        pins[11],
+        pins[12],
+        pins[13],
+        pins[14],
+        pins[15],
+        pins[16],
+        pins[17],
+    )
+rightRowPins = (pins[6], pins[7], pins[8], pins[9])
+
+
+colPins = ()
+rowPins = ()
+
+if isLeftSide:
+    colPins = leftColPins
+    rowPins = leftRowPins
+else:
+    colPins = rightColPins
+    rowPins = rightRowPins
+
+
+class KMKKeyboard(_KMKKeyboard):
+    SDA = pins[4]
+    SCL = pins[5]
+    col_pins = (
+        pins[6],
+        pins[7],
+        pins[8],
+        pins[9],
+        pins[10],
+        pins[11],
+        pins[12],
+        pins[13],
+    )
+    row_pins = (pins[17], pins[16], pins[15], pins[14])
+    diode_orientation = DiodeOrientation.COL2ROW
+    data_pin = pins[1]
+    rgb_pixel_pin = pins[0]
```

Comment:
These members are legacy only and should not be used in new contributions. These pins are to be provided to the split and rgb modules directly.